src/test/mochitest/browser_dbg-layout-changes.js

```diff
@@ -0,0 +1,84 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+/**
+ * This if the debugger's layout is correctly modified when the toolbox's
+ * host changes.
+ */
+
+"use strict";
+
+var gDefaultHostType = Services.prefs.getCharPref("devtools.toolbox.host");
+
+add_task(async function() {
+  // test is too slow on some platforms due to the number of test cases
+  requestLongerTimeout(3);
+
+  const dbg = await initDebugger("doc-iframes.html");
+  // let [tab, debuggee, panel] = await initDebugger();
+
+  // await testOrientation(dbg, );
+  // await testOrientation(dbg, );
+  // await testOrientation(dbg, );
+  //
+  // await testOrientation(dbg, "horizontal", "bottom");
+
+  const layouts = [
```

Comment:
Nit, you seem to have briefly stopped using semicolons here.

Reply:
lol